app.js

```diff
@@ -0,0 +1,8 @@
+// function () {
+//   const burger = document.querySelector('.navbar-burger');
+//   const menu = document.querySelector(`#${burger.dataset.target}`);
+//   burger.addEventListener('click', () => {
+//     burger.classList.toggle('is-active');
+//     menu.classList.toggle('is-active');
+//   });
+// }();
```

Comment:
- [ ] It appears that the JavaScript requirement outlined in the project specifications has not been fully incorporated. It is crucial to carefully review the project requirements and make sure to implement a functional Hamburger menu as well as a dynamic display of featured speakers. Your attention to these specific elements is essential for meeting the project's expectations.